src/Metrics.tcl

```diff
@@ -81,7 +81,7 @@ proc report_worst_slack_metric { args } {
     utl::error ORD 17 "both -steup and -hold specified."
   }
 
-  utl::metric_float $metric_name [worst_slack_cmd $min_max]
+  utl::metric_float $metric_name [sta::time_sta_ui [worst_slack_cmd $min_max]]
```

Comment:
should use worst_slack instead

Reply:
you'll get INF if you use the sta command - is that what you intend?

Reply:
see: https://github.com/The-OpenROAD-Project/OpenROAD/pull/2753